test/tick-processor/test-tick-processor-unknown.js

```diff
@@ -21,7 +21,7 @@ const base = require('./tick-processor-base.js');
 // Unknown checked for to prevent flakiness, if pattern is not found,
 // then a large number of unknown ticks should be present
 base.runTest({
-  pattern: /LazyCompile.*\[eval\]:1|.*%  UNKNOWN/,
+  pattern: /LazyCompile.*\[eval]:1|.*%  UNKNOWN/,
```

Comment:
`]` does not need to be escaped in a regular expression unless it occurs in a character class. I try to make a point of removing unnecessary escaping (especially in regular expressions) when I touch a file. I can put it back if you or anyone else thinks the regular expression is easier to read with it.